modules/programs/tmux.nix

```diff
@@ -147,6 +154,16 @@ in {
         description = "Base index for windows and panes.";
       };
 
+      configBeforePlugin = mkOption { 
```

Comment:
```suggestion
      extraConfigBeforePlugin = mkOption { 
```

Reply:
Done, along with changing all the respective cfg counterparts.